main.js

```diff
@@ -0,0 +1,87 @@
+const Board = function (x) {
+x = [["1", "2", "3"],
+["4", "5", "6"],
+["7", "8", "9"]];
+return x;
+}
+
+var gameBoard;
+var playerTurn;
+var moveCount = 0;
```

Comment:
This is not _that_ big of a deal for this project, but you'd normally want to refrain from using globals and writing functions with side-effects on those globals. Closures or other patterns could fix this which you would need to do if you were going to put this in a different context (like the browser) so users can't cheat by editing the code in the console.